tests/schema/test_scalars.py

```diff
@@ -99,6 +101,155 @@ def uuid_input(self, input_id: UUID) -> str:
     }
 
 
+def test_json():
+    @strawberry.type
+    class Query:
+        @strawberry.field
+        def echo_json(data: JSON) -> JSON:
+            return data
+
+        @strawberry.field
+        def echo_json_nullable(data: Optional[JSON]) -> Optional[JSON]:
+            return data
+
+    schema = strawberry.Schema(query=Query)
+
+    assert (
+        str(schema)
+        == dedent(
+            '''
+        """
+        The `JSON` scalar type represents JSON values as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf).
+        """
+        scalar JSON @specifiedBy(url: "http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf")
+
+        type Query {
+          echoJson(data: JSON!): JSON!
+          echoJsonNullable(data: JSON): JSON
+        }
+    '''  # noqa: E501
+        ).strip()
+    )
+
```

Comment:
I think this formatting in this file got screwed up by the autoformatter. It can probably be fixed